tests/headers/enum-translate-type.h

```diff
@@ -0,0 +1,14 @@
+// bindgen-flags: --translate-enum-integer-types
+
+enum my_enum1 {
+    A = 0,
+};
+enum my_enum2 {
+    B = -1,
+};
+enum my_enum3: short {
+    C = 0,
+};
+enum my_enum4: unsigned char {
+    D = -1,
+};
```

Comment:
I think this syntax is not supported by libclang 5.0 and earlier. How can I instruct CI to deal with this properly?

Reply:
Use a `.hpp` extension perhaps? I thought this syntax was specific to C++ tbh, TIL it works on C.

Reply:
> TIL it works on C

Well this may be a new Clang extension which is why it may fail on earlier versions. I've pushed a change, let's see what happens.

Reply:
CI passes now